config/settings.yml

```diff
@@ -87,3 +87,14 @@ plugins:
 
   ai_openai_api_key:
     default: ""
+
+  composer_ai_helper_enabled:
```

Comment:
We need a `composer_ai_allowed_in_pm: false` and the logic for it before merging.